insalan/tournament/models/team.py

```diff
@@ -156,6 +165,18 @@ def get_swiss_matchs(self) -> List["SwissMatch"]:
     def get_matchs(self) -> List[Union[List["GroupMatch"],List["KnockoutMatch"],List["SwissMatch"]]]:
         return self.get_group_matchs(), self.get_knockout_matchs(), self.get_swiss_matchs()
 
+    def get_seat_slot(self) -> Optional["SeatSlot"]:
+        return seat_slot.SeatSlot.objects.filter(team=self).first()
```

Comment:
team doesn't exist on TeamSlot model and you can just do self.seat_slot

Reply:
Oh right, this is probably leftover from before a refactor.